scripts/create-consolidated-report.gmp.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2021 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from uuid import UUID
+from typing import List
+from datetime import date
+from argparse import ArgumentParser, RawTextHelpFormatter
+from lxml import etree as e
+
+from gvmtools.helper import generate_uuid, error_and_exit
+
+HELP_TEXT = (
+    'This script creates a consolidated report and imports it to the GSM.'
+    ' Usable with gvm-script (gvm-tools)'
+)
+
+
+def parse_tags(tags: List):
```

Comment:
I would love to add always add a type hint for the content of the list. It is often required to read the whole function code to get the expected type.
```suggestion
def parse_tags(tags: List[str]):
```

Reply:
Like i said: Improving `gvm-tools` :)